tests/test_runtime_update_checks.py

```diff
@@ -120,7 +120,7 @@ def test_plugin_table_created(app, session):
         thread = threading.Thread(target=standalone, args=('webserver',))
         thread.daemon = True
         thread.start()
-        while thread.isAlive():
+        while thread.is_alive():
```

Comment:
Was failing with 'thread' object doesn't have isAlive method and that's true. I wonder how it passed before